### ISSUE TITLE
Some adorner elements missing aria-label and role attributes

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#961]
See the https://surveyjs.answerdesk.io/ticket/details/t5216/accessibility-all-actionable-controls-should-have-name-for-edit-move-and-delete-choice thread for more details
